spec/workers/asset_file_metadata_worker_spec.rb

```diff
@@ -18,7 +18,7 @@
 
     worker.perform(asset.id.to_s)
 
-    expect(asset.reload.last_modified.to_i).to eq(asset.last_modified_from_file.to_i)
+    expect(asset.reload.last_modified).to be_within(0.001).of(asset.last_modified_from_file)
```

Comment:
I might be tempted to add a custom `be_within_a_millisecond` matcher here to make it easier to understand the `0.001` magic number.